skyportal/schema.py

```diff
@@ -412,9 +423,10 @@ def parse_flux(self, data, **kwargs):
                        ra=data['ra'],
                        dec=data['dec'],
                        ra_unc=data['ra_unc'],
-                       dec_unc=data['dec_unc']
+                       dec_unc=data['dec_unc'],
                        )
-
+        if 'alert_id' in data and data['alert_id'] is not None:
+            p.alert_id = data['alert_id']
```

Comment:
Simply do
`p.alert_id = data.get('alert_id', data['mjd'])`
instead of the if statement? (mjd is required, right?)